android/source/VideoLocker/res/anim/no_transition.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android" >
+
+    <translate
+        android:duration="500"
```

Comment:
This is probably just a basic android question, but why does ``no_transition`` take half a second? Shouldn't it be instant?